src/applications/representative-form-upload/components/ConfirmationPageView.jsx

```diff
@@ -1,21 +1,32 @@
 import React, { useEffect, useRef } from 'react';
 import { format } from 'date-fns';
+import { useSelector } from 'react-redux';
 
 import { scrollTo } from 'platform/utilities/scroll';
 import { waitForRenderThenFocus } from 'platform/utilities/ui/focus';
 
 export const ConfirmationPageView = ({
   submitDate,
-  confirmationNumber,
+  benefitType,
   childContent = null,
 }) => {
+  const form = useSelector(state => state.form || {});
+  const { first, last } = form.data.veteranFullName;
+  const { city, state, postalCode } = form.data.address;
   const alertRef = useRef(null);
 
   const formattedSubmitDate =
     submitDate && typeof submitDate === 'object'
       ? format(submitDate, 'MMMM d, yyyy')
       : null;
 
+  const formattedSubmitDateTime =
+    submitDate && typeof submitDate === 'object'
+      ? format(submitDate, "MMMM d, yyyy 'at' h:mm aaaa 'ET'")
```

Comment:
I remember hearing that ITFs are always submitted in ET, so I hardcoded it here. Haven't been able to confirm though. Also not sure if the api would always respond with the time in ET. Wondering if anyone else had confirmation.